syntax = "proto3";

package NSound.NServices;

import "protos/client-message.proto";
import "protos/server-message.proto";


// gRPC sound service
service SoundStreamRouter {

    // main channel for communication
    rpc RouteStream(stream NSound.TClientMessage) returns (stream NSound.TServiceMessage) {}

}